src/components/PracticeSection/MultipleChoice.js

```diff
@@ -249,7 +252,10 @@ export default function MultipleChoice() {
           student.add("reward_badge_ids", reward);
           const points = student.get("total_points");
           const rewardPoints = points + 50;
+          const originalCoins = student.get("coins");
+          const rewardCoins = originalCoins + 25;
```

Comment:
I know this is inherited code, but it would be nice to extract this literal to a constant and put it at the top of the file; e.g. `const CORRECT_ANSWER_COIN_REWARD = 25` or something like that